sdk/typescript/src/types/sui-bcs.ts

```diff
@@ -433,6 +449,16 @@ const BCS_0_23_SPEC = {
   },
 };
 
+const BCS_0_24_SPEC = {
+  ...BCS_0_23_SPEC,
```

Comment:
```suggestion
  ...BCS_SPEC,
```

This should be `BCS_SPEC` because the new `MoveCallTx` has been supported in 0.24.0